templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -425,28 +425,26 @@
         "duplicatedAssets.$": "$.Payload.duplicatedAssets"
       },
       "ResultPath": "$.WorkflowResult",
-      "Next": "Check workflow status and get Succeeded, Failed and Duplicated asset ids"
+      "Next": "Check workflow status"
     },
-    "Check workflow status and get Succeeded, Failed and Duplicated asset ids": {
+    "Check workflow status": {
```

Comment:
I just realised that this step and the previous one weren't originally given the most accurate names
   1. Maybe the previous step could be (in order to emphasise that we are returning ids)  "Get workflow status & asset id per status" or "Get workflow & asset id statuses"
   2. and then this step could be named "Check for Failed or Succeeded workflow status" or "Check workflow status for Failed or Succeeded" (even though you could get that from reading the step definition, I was thinking from a non-technical person's perspective)?

Reply:
I don't think we're going to get many non-technical people reading our step functions and even if there were, "Check workflow status" conveys broadly what it's doing. I would prefer to keep the names short and as you say, the detail can be found in the step function details.